recaf-ui/src/main/java/me/coley/recaf/scripting/Script.java

```diff
@@ -38,7 +38,8 @@ private Script(String source, boolean isFile) {
         if(isFile) {
             try {
                 parseTags();
-            } catch (IOException ignored) {
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
This would be logging directly to std-err. We want to create a logger so that the rest of Recaf can intercept the issue.

`private static final Logger logger = Logging.get(Script.class);`

and

`logger.error("Failed to do bla", e);`